lcc_lokavaluto_app_connection/tests/test_partner_services_wallet_restriction_rules.py

```diff
@@ -139,7 +142,7 @@ def test_rule_allowing_all_senders(self):
             {
                 "name": "Test Rule",
                 "sender_wallet_domain": "[]",
```

Comment:
Le test que j'avais écrit ne représente pas bien la réalité de ce qui se passe lorsqu'on omet de préciser un sender-wallet_domain depuis l'IHM. Il faut modifier comme cela

```suggestion
                "sender_wallet_domain": None,
```

Ce qui fait planter le test avec `ValueError: "'bool' object has no attribute 'strip'" while compiling`. Je propose une correction dans ta PR pour éviter les conflits.

Reply:
(vu avec Stephan) je propose une PR et il corrigera les conflits.

PR : https://github.com/Lokavaluto/lokavaluto-addons/pull/362/files